commodore/config.py

```diff
@@ -59,6 +50,19 @@ def config_file(self):
     def default_component_base(self):
         return f"{self.global_git_base}/commodore-components"
 
+    @property
+    def api_token(self):
+        return self.__api_token
+
+    @api_token.setter
+    def api_token(self, api_token):
+        if api_token is not None:
+            p = P(api_token)
+            if p.is_file():
+                with open(p) as apitoken:
+                    api_token = apitoken.read()
+            self.__api_token = api_token.strip()
```

Comment:
Usually a single underscore is used to mark a field "private", double underscores are handled specially to allow "class private" variables, cf. https://docs.python.org/3/tutorial/classes.html#private-variables

Here, a single underscore should be sufficient